src/cmakeTools.ts

```diff
@@ -1495,79 +1477,79 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
             return 0;
         }
 
-        const config_retc = await this.ensureConfigured();
-        if (config_retc === null) {
+        const configRetc = await this.ensureConfigured();
+        if (configRetc === null) {
             throw new Error(localize('unable.to.configure', 'Build failed: Unable to configure the project'));
-        } else if (config_retc !== 0) {
-            return config_retc;
+        } else if (configRetc !== 0) {
+            return configRetc;
         }
         drv = await this.getCMakeDriverInstance();
         if (!drv) {
             throw new Error(localize('driver.died.after.successful.configure', 'CMake driver died immediately after successful configure'));
         }
-        let targets = targets_;
+        let newTargets = targets;
```

Comment:
This gets immediately reassigned, so it's better to initialize it to undefined, I think.

Reply:
Can you make a note and we'll fix it in the next PR since this one is just formatting?